src/js/views/globals/search/patient-search.scss

```diff
@@ -33,12 +33,14 @@
 
 .patient-search__header {
   border-bottom: 1px solid $black-90;
+  margin: 0 8px;
+  padding-left: 16px;
```

Comment:
The design doc includes `8px` of spacing between the input bottom border on both sides. This adds that and updates the left padding for aligning with the help text and results content.

<img width="578" alt="Screen Shot 2022-09-08 at 12 03 56 PM" src="https://user-images.githubusercontent.com/35355575/189182872-b3acaaa4-18f8-4af4-b754-b921f94267c6.png">